### PR TITLE
docs: adds quick start example via snooze

### PR DESCRIPTION
**Pull request by waynevanson**
Adds a quick start example using `snooze` and `runit`.

No hard feelings if you don't want to merge this.
I'll add it somewhere else because I feel it's useful, predominately for beginners.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,26 @@ will not build the package files until they are updated upstream. To manually
...
+# Download `xbps-mini-builder` from github via git
+cd <directory>
+git clone https://github.com/the-maldridge/xbps-mini-builder.git
+
+# Add packages to list, separated by newline
+echo "package_one\npackage_two" >> <directory>/packages.list
+
+# Create the service and add the snooze command to the run file
+mkdir /etc/sv/<service_name>
+echo "snooze -d* <directory>/xbps-mini-installer" > /etc/sv/<service_name>/run
```
Comment (the-maldridge) [inline README.md:46]: This should have an exec, should run as a different user, and `-d*` as it is a default value for snooze.

File: README.md [hunk 2]
```diff
@@ -25,3 +25,26 @@ will not build the package files until they are updated upstream. To manually
...
+```bash
+# Assuming runit is installed, download snooze
+xbps-install -y snooze
+
+# Download `xbps-mini-builder` from github via git
+cd <directory>
+git clone https://github.com/the-maldridge/xbps-mini-builder.git
+
+# Add packages to list, separated by newline
+echo "package_one\npackage_two" >> <directory>/packages.list
```
Comment (the-maldridge) [inline README.md:42]: This should be a heredoc for clarity, or a direction to use $editor.

File: README.md [hunk 3]
```diff
@@ -25,3 +25,26 @@ will not build the package files until they are updated upstream. To manually
...
+
+# Create the service and add the snooze command to the run file
+mkdir /etc/sv/<service_name>
+echo "snooze -d* <directory>/xbps-mini-installer" > /etc/sv/<service_name>/run
+
+# Add the service to runit
```
Comment (the-maldridge) [inline README.md:48]: There should be a log symlink to vlogger prior to enabling the service.

**Review by the-maldridge — CHANGES_REQUESTED** [review]
Ultimately a lot of what you're wanting to add as documentation should probably just be summed up by packaging the mini-builder.  As it has a limited use case I have not pursued this path, but your quickstart as written misses several important points in running the builder securely.

**Comment by waynevanson** [comment]
@the-maldridge  Thank you for your feedback.

Packaging the mini builder sounds like a good idea. 

Once it's packaged, then I'll come back and edit this.

**Pull request closed without merging**